### PR TITLE
Update to hardcoded paths accessing the NuMI flux systs file.

### PR DESCRIPTION
**Pull request by apwood-physics**

### CODE REVIEW

**Comment by brucehoward-physics** [comment]
It looks like most of the actual substantive changes are just locations/names of products inside the functions that go get stuff from the ROOT file? Did anything else substantive change?

(Note: curious if you ran a style macro over this or something? A lot of small changes here and there don't seem substantive, hence my questions.)

**Comment by apwood-physics** [comment]
> It looks like most of the actual substantive changes are just locations/names of products inside the functions that go get stuff from the ROOT file? Did anything else substantive change?
> 
> (Note: curious if you ran a style macro over this or something? A lot of small changes here and there don't seem substantive, hence my questions.)

That’s correct, I didn’t make any changes other than pointing the paths to the appropriate places in the root file. 

Also, yes, although it was unintentional. I have my editor configured to run clang_format automatically on write.

File: sbnana/CAFAna/Systs/NuMIFluxSysts.cxx [hunk 1]
```diff
@@ -11,157 +11,152 @@
...
+  std::vector<const ISyst *> ret;
+  for (std::string name : syst_names)
+    ret.push_back(GetNuMIFluxSyst("fractional_uncertainties/hadron/" + name,
+                                  "hfrac_hadron_", name));
+  return ret;
+}
+
+//----------------------------------------------------------------------
+std::vector<const ISyst *> GetNuMIBeamlineFluxSysts() {
+  const std::vector<std::string> syst_names = {
```
Comment (brucehoward-physics) [inline sbnana/CAFAna/Systs/NuMIFluxSysts.cxx:134]: Is beam_shift_x purposefully left out?

File: sbnana/CAFAna/Systs/NuMIFluxSysts.cxx [hunk 2]
```diff
@@ -11,157 +11,152 @@
...
+  const std::string key = dir + "/" + prefix + name;
+  if (cache.count(key) == 0)
+    cache[key] = new NuMIFluxSyst(dir, prefix, name);
+
+  return cache[key];
+}
+
+//----------------------------------------------------------------------
+std::vector<const ISyst *> GetNuMIHadronProductionFluxSysts() {
+  const std::vector<std::string> syst_names = {
```
Comment (brucehoward-physics) [inline sbnana/CAFAna/Systs/NuMIFluxSysts.cxx:121]: I _believe_ there are some discrepancies here?

* `att` vs `attenuation`
* pCQEL missing?

**Comment by apwood-physics** [comment]
@miquelnebot, I'm closing this PR to fix the issues @brucehoward-physics spotted with the `attenuation` and `pCQEL` systematics.

**Pull request closed without merging**